app/workers/reset_project_counters_worker.rb

```diff
@@ -0,0 +1,39 @@
+class ResetProjectCountersWorker
+  include Sidekiq::Worker
+
+  sidekiq_options queue: :data_high
+
+  def perform(project_id)
+    project = Project.find(project_id)
+
+    return unless project.launch_date
```

Comment:
I was thinking they probably only want to reset when they go live and thus have a launch date. If we have some other date in this field that's not the launch date, I think we'd want to find a different name for the column while we still easily can.